examples/extend.py

```diff
@@ -0,0 +1,17 @@
+from pylab import *
+from pyspecdata import *
+from numpy.random import normal, seed
+from matplotlib.ticker import NullLocator, MultipleLocator, FormatStrFormatter
+seed(7919)
+d = nddata(normal(size=10000)+1j*normal(size=10000), [100,100], ['y','x']).labels(
+        {'x':r_[0:1:100j],
+        'y':r_[0:0.1:100j]})
+with figlist_var() as fl:
+    fl.next('random data')
+    fl.image(d)
+    d.extend('x',1.5)
+    fl.next('extend along $x$')
+    fl.image(d)
+    d.extend('y',-0.05,fill_with=1)
+    fl.next('extend along $y$')
+    fl.image(d)
```

Comment:
so, I added this example, whih I really thought should work, but it gives me some type of type error.  Can you edit either the example or the function?

Reply:
This currently works